src/consensus/forging.md

```diff
@@ -0,0 +1,84 @@
+# Forging new blocks
+
+Forging new blocks is the process of packing data that has not yet been included in the
+blockchain (pending transactions) into a fresh block. Blocks are signed by stake pool
+credentials, so only the nodes which are configured with such credentials strictly need to
+implement the forging mechanism.
+
+## Forging
+
+Every slot, the stake pool (whose keys the node was configured with) might be
```

Comment:
Should we mention "stake pool" or simply "nodes"?

Reply:
"Nodes" would we wrong. If a stake pool configures several nodes with the keys all of them will try to create a block (only one will be accepted by the network though)